src/docs/flex-cli/manage-search-schemas-with-flex-cli/index.md

```diff
@@ -111,6 +118,14 @@ flex-cli search set --key category --type enum --scope public -m my-test-marketp
 flex-cli search set --key amenities --type multi-enum --scope public -m my-test-marketplace
 ```
 
+Note that we don't need to supply `--schema-for` option when we add schema for `listing` as `listing` is the default.
+
+We can also add a search schema for `userProfile` with `long` type.
```

Comment:
The current section, under which this text is, is written more like a concrete tutorial for adding listing search schema. As such, this text doesn't make sense there, because the operator is not interested in adding userProfile schema at that moment. This could perhaps be a new heading "Adding user search schema".

Note that the opening paragraphs in this article also present it as tutorial for managing listing search schema. So some rephrasing is needed.

Reply:
* opening paragraphs changed
* new heading "Adding user search schema" added
* existing heading renamed "Adding schemas" -> "Adding listing search schemas"